controllers/comments.controllers.js

```diff
@@ -0,0 +1,9 @@
+const { deleteCommentById } = require("../models/comments.models");
+
+exports.destroyCommentById = (req, res, next) => {
```

Comment:
Love the function name 😆

Reply:
Haha thanks, it's meant to be keeping in line with RESTful controllers and handily it doesn't compete with the name of the model delete method.  😆